connectors/src/lib/data_sources.ts

```diff
@@ -1148,6 +1152,7 @@ export async function upsertFolderNode({
     folderId,
     timestampMs ? timestampMs : now.getTime(),
     title,
+    parentId ?? null,
```

Comment:
do we send null even if parents[1] is set?

Reply:
done